arch/arc/core/prep_c.c

```diff
@@ -119,8 +119,8 @@ void _PrepC(void)
 {
 	_icache_setup();
 	adjust_vector_table_base();
-	_bss_zero();
-	_data_copy();
-	_Cstart();
+	z_bss_zero();
+	data_copy();
```

Comment:
z_data_copy()?

Reply:
data_copy() comes from the previous kernel patch in file kernel/include/kernel_internal.h:
```
 #ifdef CONFIG_XIP
-void _data_copy(void);
+void data_copy(void);
 #else
-static inline void _data_copy(void)
+static inline void data_copy(void)
```
Coccinelle sees the static inline void _data_copy() and correctly renames it data_copy(). Of course the above wants it the opposite way.

Reply:
Fixed.